### PR TITLE
LIVE-1173: remove old btt trc10

### PR DESCRIPTION
**Pull request by henri-ly**
https://ledgerhq.atlassian.net/browse/LIVE-1173

Remove BTT and rename it (token 1002000) from the add token list
also fix the script of import token and

### CODE REVIEW

File: script/crypto-assets-importer/tron/trc10-tokens.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 var whitelist = [
-  1002000, // BTT
+  // 1002000, // BTT
```
Comment (haammar-ledger) [inline script/crypto-assets-importer/tron/trc10-tokens.js:2]: Is that comment expected, shouldn't it be removed?
(Genuine question, I have no idea how it's supposed to work. :smile: )

**Review by ghost — COMMENTED** [review]
Questions about the changes in the `trc10.js` file:
- Why the change of IDs from string to number?
- It doesn't seem the BTT token is removed from the file? (2 different instances btw... I'm not even sure which one should be removed)
- Seems there's an addition at the end of a "TRON"/"TRX" token, is this expected?

**Comment by henrily-ledger** [comment]
- It's seems that we used to have r.id as a string but it somehow become a number, so the compare couldn't match since it's number vs string. So i put everything as a string to be simplier
- Well we need to have BTT accessible to the old owner but just remove the possibility to add BTT new token (the comment from the list remove it from the whitelist of possible added token)
- I have no idea. Script generate himself from trongrid API. So I guess it's should be expected ?

File: script/crypto-assets-importer/tron/trc10-tokens.js (hunk 1 above)
Comment (henrily-ledger) [reply script/crypto-assets-importer/tron/trc10-tokens.js:2]: I'll remove it

**Review by ghost — APPROVED** [review]

**Pull request merged**